Microsoft.Toolkit.Uwp.UI.Controls/GridSplitter/GridSplitter.Helper.cs

```diff
@@ -30,41 +30,105 @@ private static bool IsStarRow(RowDefinition definition)
             return ((GridLength)definition.GetValue(RowDefinition.HeightProperty)).IsStar;
         }
 
-        private void SetColumnWidth(ColumnDefinition columnDefinition, double horizontalChange, GridUnitType unitType)
+        private bool SetColumnWidth(ColumnDefinition columnDefinition, double horizontalChange, GridUnitType unitType)
         {
             var newWidth = columnDefinition.ActualWidth + horizontalChange;
+
+            var minWidth = columnDefinition.MinWidth;
+            if (minWidth != double.NaN && newWidth < minWidth)
```

Comment:
minWidth != double.NaN will always be true, please use double.IsNaN.
Please also apply this comment of the rest of comparisons using Nan